src/include/duckdb/common/helper.hpp

```diff
@@ -18,14 +18,14 @@
 namespace duckdb {
 #if !defined(_MSC_VER) && (__cplusplus < 201402L)
 template <typename T, typename... Args>
-unique_ptr<T> make_unique(Args &&...args) {
+unique_ptr<T> make_unique(Args &&... args) {
```

Comment:
Please don't check in this change, its a flip-flop between versions of clang-format

Reply:
There is another one or two like it

Reply:
I agree, of course, I can look at it, but of course, this won't solve the underlying problem. 
Maybe enforcing a clang-format version is an option?

Reply:
I just don't add those particular changes to my commit

Reply:
Let's just exclude these three files from the make format-fix?